WordPress/Classes/ViewRelated/Reader/ReaderStreamViewController.swift

```diff
@@ -1086,6 +1091,30 @@ import WordPressComAnalytics
         }
     }
 
+    func backgroundFetch(_ completionHandler: ((UIBackgroundFetchResult) -> Void)?) {
+        print("=========>>>>>>>>>>> backgroundFetch")
+        print("=========>>>>>>>>>>> \(String(describing: readerTopic?.title))")
+
+        let lastSeenPostID = (tableViewHandler.resultsController.fetchedObjects?.first as? ReaderPost)?.postID
```

Comment:
How would you suggest doing that? I want to continue even if I didn't have any content loaded before, right? Even though I can not think of any reasonable circumstance for that to happen...